pipelines/build/common/build_base_file.groovy

```diff
@@ -797,8 +821,12 @@ class Builder implements Serializable {
                                                 )
 
                                                 // cache Linux platform
-                                                if (configuration.key.toLowerCase().contains('linux')) {
-                                                    linuxTargets.add(config.ARCHITECTURE)
+                                                if (config.TARGET_OS.equals('linux')) {
+                                                    if (config.VARIANT.equals('openj9') && (config.ADDITIONAL_FILE_NAME_TAG && !config.ADDITIONAL_FILE_NAME_TAG.endsWith('XL'))) {
+                                                        taggedLinuxTargets.add(config.ARCHITECTURE)
+                                                    } else {
+                                                        linuxTargets.add(config.ARCHITECTURE)
```

Comment:
Can you add a comment about what `taggedLinuxTargets` and `linuxTargets` are? It appears tagged is for Certified edition because of the additionalFileNameTag. Which tells me this allows us to build Certified and Open in the same build. Are we assuming too much about what additionalFileNameTag could/usually infer?

Reply:
done